app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,19 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
giải thích hoạt động của line code này nào

Reply:
Toán tử ||= là để kiểm tra rồi gán giá trị phù hợp. Nếu `@current_user` = `nil` hoặc `false` thì sẽ dc gán giá trị mới hoặc nếu nó được gán `id` rồi thì giá trị `id` đó k thay đổi